.github/workflows/Rebuild Database Templates.yml

```diff
@@ -0,0 +1,640 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Rebuild Database Templates
+
+on:
+  workflow_dispatch:
+
+env:
+  INFORMATIONAL_VERSION: "7.0"
+  BUILD_INCREMENTER: "1"
+  CONFIGURATION: "Release"
+  VERSION_MAJOR: "7"
+  VERSION_MINOR: "0"
+  HEAD_REF:  ${{ GITHUB.HEAD_REF }}
+  REF_NAME:  ${{ GITHUB.REF_NAME }}
+  REPOSITORY_DISPATCH_BRANCH: ${{ github.event.client_payload.branch }}  
+  EDFI_ODS_TOKEN: ${{ secrets.REPO_DISPATCH_TOKEN }}
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Setup .NET
+      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
+      with:
+        dotnet-version: 6.0.x
+    - name: Checkout Ed-Fi-ODS-Implementation
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
+          path: Ed-Fi-ODS-Implementation/
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+           ./build.githubactions.ps1 CheckoutBranch -RelativeRepoPath "."
+    - name: Checkout Ed-Fi-ODS
```

Comment:
Yes  I did . Thanks

Reply:
I can't removed ODS repo Checkout steps because Import-Module "./logistics/scripts/modules/settings/settings-management.psm1" which internally uses logistics/scripts/modules/load-path-resolver.ps1  . So path resolver needs to have ODS repo for sure .  So I added it back now